src/routes/_components/LengthIndicator.html

```diff
@@ -42,11 +41,11 @@
     store: () => store,
     computed: {
       lengthToDisplay: ({ length, max }) => max - length,
-      lengthLabel: ({ overLimit, lengthToDisplayDeferred }) => {
-        if (overLimit) {
-          return formatIntl('intl.overLimit', { count: -lengthToDisplayDeferred })
+      lengthVerbosity: ({ lengthToDisplayDeferred }) => {
+        if (lengthToDisplayDeferred > 10) {
+          return 'off'
         } else {
-          return formatIntl('intl.underLimit', { count: lengthToDisplayDeferred })
+          return 'polite'
```

Comment:
Now that `intl.overLimit` and `intl.underLimit` are unused, we can remove them from `en-US.js` (and the other intl files in the same directory).